site/content/docs/5.0/components/navs-tabs.md

```diff
@@ -1,8 +1,9 @@
 ---
 layout: docs
-title: Navs
+title: Navs and tabs
 description: Documentation and examples for how to use Bootstrap's included navigation components.
 group: components
+aliases: "/docs/5.0/components/navs/"
```

Comment:
I don't think we need this alias since v5 is a major version. I need to remember to drop any redundant redirects when we hist stable.

Reply:
might still be nice to keep around in case people come from older versions and just hacked the url with `5.0`? not precious about it either way though

Reply:
We always followed this pattern when we released major/minor versions since the URLs change. I'd rather keep things clean because we can easily end up with hundreds of redirects.